packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -85,8 +87,12 @@ function updateNodeStyle(node: NodeOrText) {
   });
 }
 
-// If an item has a specified icon, that icon will be used for it and for subsequent
-// items at that level to prevent duplication of icons attribute declarations.
+// If an item has a specified icon, that icon will be saved and used for it and for
+// subsequent items at that level to prevent duplication of icons attribute declarations.
+// However, if an item has a specified icon and it is one-off, the icon will only be
+// used for that item.
+// Items with one-off icons do not overwrite the previously saved icon, meaning that
+// subsequent items will still use the last saved icon.
```

Comment:
Need to correct the comment here to show that other attributes also won't get inherited.